.github/workflows/ci.yml

```diff
@@ -25,8 +27,15 @@ jobs:
 
       - name: Install e2e deps
         run: |
-          go get -u github.com/prometheus/promu@v0.11.1
-          go get -u github.com/josephspurrier/goversioninfo/cmd/goversioninfo@v1.2.0
+          # promtool brings in a lot of dependencies, slowing down compilation via `go get` and `go install`, so
+          # download the binary directly and extract.
+          # go get github.com/prometheus/prometheus/cmd/promtool@main
+          Invoke-WebRequest -Uri https://github.com/prometheus/prometheus/releases/download/v$($Env:PROMTOOL_VER)/prometheus-$($Env:PROMTOOL_VER).windows-amd64.zip -OutFile prometheus-$($Env:PROMTOOL_VER).windows-amd64.zip
+          Expand-Archive -Path prometheus-$($Env:PROMTOOL_VER).windows-amd64.zip -DestinationPath .
+          Copy-Item -Path prometheus-$($Env:PROMTOOL_VER).windows-amd64\promtool.exe -Destination "$(go env GOPATH)\bin"
+
+          go install github.com/prometheus/promu@v0.11.1
```

Comment:
It would be nice to make promu version a variable, and update to 0.13.0. See [here](https://github.com/prometheus/promu/releases/tag/v0.13.0)

Reply:
Good idea, I've add that in another commit